### PR TITLE
TUNIC: Logic for bushes in guard house 2 upper and belltower

### PR DESCRIPTION
**Pull request by ScipioWright**
## What is this fixing or adding?
I <3 bushes
Adds logic for breaking the bushes in guard house 2 upper. Only relevant with ladder shuffle or ER or whatever else uses the ER rules on.
Also forest belltower.

## How was this tested?
Test gens.

### CODE REVIEW

**Comment by Seatori** [comment]
Forest Belltower Main behind bushes => Forest Belltower Main is also possible with Magic Orb + Magic Dagger. Should probably be in easy ice grapple logic.

File: worlds/tunic/er_rules.py [hunk 1]
```diff
@@ -403,6 +409,13 @@ def set_er_region_rules(world: "TunicWorld", regions: Dict[str, Region], portal_
         connecting_region=regions["Forest Belltower Lower"],
         rule=lambda state: has_ladder("Ladder to East Forest", state, world))
 
+    regions["Forest Belltower Main behind bushes"].connect(
+        connecting_region=regions["Forest Belltower Main"],
+        rule=lambda state: can_get_past_bushes(state, world))
```
Comment (Seatori) [inline worlds/tunic/er_rules.py:414]: This is also possible with Magic Orb + Magic Dagger by waiting for a slime to come over to you, using the dagger to freeze it, and then grappling it with the orb. Should probably require easy ice grapple logic.

**Review by roseasromeo — APPROVED** [review]
Added logic is in line with how the rando functions, new regions prevent soft-locking due to lack of access to bush-cutting.

**Comment by Seatori** [comment]
Tested in-game and verified that all of the listed methods were valid ways of reaching each entrance.
Also ran several test generations using plando to verify that each method was correctly registered, and viewed playthroughs in which each method was in play.
Lastly, I verified that the easy ice grappling method wasn't expected with ice grappling disabled by using plando to create an impossible seed without using Magic Orb + Magic Dagger on that entrance, seeing that it failed with ice grappling disabled, and that it succeeded when it was enabled on all three difficulty levels.

**Review by Exempt-Medic — APPROVED** [review]

**Pull request merged**